include/autoware_state_machine/autoware_state_machine.hpp

```diff
@@ -180,6 +180,8 @@ class AutowareStateMachine : public rclcpp::Node
   void setEngageProcess(bool request, bool accept);
   std::pair<bool, bool> getEngageProcess(void);
   bool waitingForEngageAccept(void);
+  std::pair<std::string, double> getNearestStopReason(
```

Comment:
関数内処理で距離以外の優先度も扱うので、それがわかる命名としたいです。
例えば下記のような命名でいかがでしょう？
```suggestion
  std::pair<std::string, double> getNearestStopReasonWithPriority(
```

Reply:
修正しました